scripts/jenkins/cloud/ansible/bootstrap-vcloud-nodes.yml

```diff
@@ -77,13 +77,16 @@
         # using mitogen through bastion hosts.
         - name: Wait for cloud nodes to be accessible
           delegate_to: localhost
-          wait_for:
```

Comment:
using wait_for instead of wait_of_connection is temporary used(manual cloud couldn't connect through bastion host - this gave my better connection stability to test it)  and before the merge will be removed